src/chi/components/sidebar/sidenav.scss

```diff
@@ -116,6 +116,16 @@ $sidenav-lg-width: 12.5rem;
       &:not(.-active) > a:hover {
         background-color: set-color(grey, 10);
       }
+
+      .m-sidenav__drawer-item-list {
+        display: none;
+        overflow: hidden;
+        transition: height 0.075s ease-out;
+
+        &.visible {
```

Comment:
@janibolkvadze we usually use - (dash) for this kind of classes (ie: -active). Please add it.